mobile/library/jni/jni_utility.cc

```diff
@@ -315,7 +314,7 @@ void javaByteArrayToProto(JniHelper& jni_helper, jbyteArray source,
 }
 
 LocalRefUniquePtr<jbyteArray> protoToJavaByteArray(JniHelper& jni_helper,
-                                                   const Envoy::Protobuf::MessageLite& source) {
```

Comment:
this is the bit I'm worried about causing problems for import

Reply:
FWIW builds and passes without proto lite build turned on so I _think_ it's Ok

Reply:
Yeah, I believe the Java and Kotlin tests can work with either proto lite or full proto.